### PR TITLE
[Reviewer: Ellie] Fixes for https://github.com/Metaswitch/clearwater-cassandra/issues/87

### PR DESCRIPTION
**Pull request by rkd-msw**
Basically as suggested in #87.

### CODE REVIEW

**Comment by eleanor-merry** [comment]
LGTM

**Comment by rkd-msw** [comment]
@JimBunch - this fixes #87, which has a significant impact. I believe Doug has already made this change in the code you're testing with, so this just ensures it's preserved in future builds. Are you happy for me to treat #87 as critical and merge this fix?

File: clearwater-cassandra/usr/share/clearwater/clearwater-cluster-manager/plugins/cassandra_plugin.py [hunk 1]
```diff
@@ -173,7 +173,8 @@ def write_new_cassandra_config(self, seeds_list, destructive_restart=False):
             run_command("chown -R cassandra /var/lib/cassandra")
 
         self.start_cassandra()
-        os.remove("/etc/clearwater/force_cassandra_yaml_refresh")
+        if os.path.exists("/etc/clearwater/force_cassandra_yaml_refresh"):
+            os.remove("/etc/clearwater/force_cassandra_yaml_refresh")
```
Comment (bossmc) [inline clearwater-cassandra/usr/share/clearwater/clearwater-cluster-manager/plugins/cassandra_plugin.py]: It's more pythonic (and generally better) to catch the OSError and ignore it if `e.errno == errno.ENOENT`.

https://docs.python.org/2/glossary.html#term-eafp

**Comment by rkd-msw** [comment]
@JimBunch - I haven't seen a response from you on this. Should I be merging this fix?

**Comment by JimBunch** [comment]
Please merge to the fixes branch. Thanks.

**Pull request merged**